core/qthelper.cpp

```diff
@@ -1399,7 +1399,7 @@ QString getUUID()
 		uuidString = uuid.toString();
 		qPrefUpdateManager::set_uuidString(uuidString);
 	}
-	uuidString.replace("{", "p").replace("}", "");
+	uuidString.replace("{", "").replace("}", "");
```

Comment:
This seems out of place. Leftover from something else?

Reply:
Actually. no, this is a cleanup of something that I merged earlier by mistake.
So yes, unrelated, but definitely the right thing to do.

Reply:
Break it out to a separate commit, saying that, because having it here in this commit makes no sense.

Reply:
done